Proposed clarifications regarding replyUrls (#90778)
Azure/azure-content: articles/active-directory-b2c/tutorial-register-applications.md

    The following restrictions apply to redirect URIs:

>     * The reply URL must begin with the scheme `https`, unless using localhost.

Reviewer: ```suggestion
    * The reply URL must begin with the scheme `https`, unless you use a localhost redirect URL.
```

Author: I copied the wording from the "Applications (Legacy)" blade thinking it looked better if they were in sync. Specifying "localhost redirect URL" is more specific though, so maybe that should be carried out in both places? (Though I'm guessing the legacy blade will eventually go away.)

Reviewer: Oooh, I see. You can commit my suggested change so that I can sign off the PR. Thanks.

    * The reply URL is case-sensitive. Its case must match the case of the URL path of your running application. For example, if your application includes as part of its path `.../abc/response-oidc`,  do not specify `.../ABC/response-oidc` in the reply URL. Because the web browser treats paths as case-sensitive, cookies associated with `.../abc/response-oidc` may be excluded if redirected to the case-mismatched `.../ABC/response-oidc` URL.

>     * The reply URL should include or exclude the trailing forward slash as your application expects it. For example, `https://contoso.com/auth-response` and `https://contoso.com/auth-response/` may be treated as non matching URLs in your application.

Reviewer 2: ```suggestion
    * The reply URL should include or exclude the trailing forward slash as your application expects it. For example, `https://contoso.com/auth-response` and `https://contoso.com/auth-response/` might be treated as nonmatching URLs in your application.
```
Minor style edits

    * The reply URL must begin with the scheme `https`, unless using `localhost`.
    * The reply URL is case-sensitive. Its case must match the case of the URL path of your running application. For example, if your application includes as part of its path `.../abc/response-oidc`,  do not specify `.../ABC/response-oidc` in the reply URL. Because the web browser treats paths as case-sensitive, cookies associated with `.../abc/response-oidc` may be excluded if redirected to the case-mismatched `.../ABC/response-oidc` URL.

>     * The reply URL should include or exclude the trailing forward slash as your application expects it. For example, `https://contoso.com/auth-response` and `https://contoso.com/auth-response/` may be treated as non matching URLs in your application.

Reviewer 2: ```suggestion
    * The reply URL should include or exclude the trailing forward slash as your application expects it. For example, `https://contoso.com/auth-response` and `https://contoso.com/auth-response/` might be treated as nonmatching URLs in your application.
```
Style edits
